roles/rsyslog/tasks/deploy.yaml

```diff
@@ -1,20 +1,23 @@
 ---
 - name: Install/Update required packages
   package:
-    name: "{{ rsyslog_role_packages }}"
+    name: "{{ __rsyslog_packages }}"
     state: latest
   when:
-    - not use_rsyslog_image | default(false) |bool
+    - not rsyslog_in_image | default(false) | bool
```

Comment:
why `default(false)` if you have already set the default in defaults?

Reply:
Fixed.

Reply:
@nhosoi there are some remaining occurrences.